src/client/components/FormGenerator.tsx

```diff
@@ -58,8 +58,9 @@ const FormGenerator: React.FC<FormGeneratorProps> = ({
           />
         </FormControl>
       ))}
-      {relations.map((relation) => (
+      {relations.map((relation) => relation.showInForm && (
         <RelationSwitch
+          key={relation.name}
```

Comment:
Not sure if names can be repeated here but if that could be the case I advice using the index on the map loop.

Reply:
Those names are unique because they come from property names of a BaseEntity (typeorm) class. But its a good observation, thanks!